pages/your_tasks/ethics.md

```diff
@@ -0,0 +1,116 @@
+---
+title: Ethical aspects
+contributors: [Karel Berka, Korbinian Bösl, Erin Calhoun, Paulette Lieby]
+description: Working on aspects in the management of research data that can raise ethical issues
+page_id: ethics
+related_pages:
+  tool_assembly: [covid-19, transmed, tsd, csc]
+training:
+- name: 'Ethics/ELSI considerations - From FAIR to fair data sharing - recorded webinar'
+  registry: TeSS
+  url: https://tess.elixir-europe.org/materials/ethics-elsi-considerations-from-fair-to-fair-data-sharing
+- name: 'Biomedical data: Ethical, legal and social implications - e-learning'
+  registry: TeSS
+  url: https://tess.elixir-europe.org/materials/biomedical-data
+- name: 'From Swab to Server: Testing, Sequencing, and Sharing During a Pandemic - free online course'
+  registry: TeSS
+  url: https://tess.elixir-europe.org/materials/from-swab-to-server-testing-sequencing-and-sharing-during-a-pandemic
+---
+​
+Ethical issues refer to moral principles and standards that guide human conduct and define what is considered right or wrong within a particular context.
```

Comment:
Ethics refers to moral principles and norms that help us identify right from wrong within a particular context. Ethical issues/concerns typically arise when these principles conflict. Navigating through such concerns often requires one to compare the benefits of an action with the (harmful) consequences of doing so.